[{"Owner":"mandim","Date":"2018-04-03T08:09:26Z","Content":"_lt_div class_eq__qt_mages_qt__gt_\n\t\t\t\n_lt_p_gt_\n\tHello_co_\n_lt_/p_gt_\n\n_lt_p_gt_\n\tI have just updated the last version of blender exporter and I_t_m not able to export a simple scene.\n_lt_/p_gt_\n\n_lt_p_gt_\n\tThis is the error log I get_dd_\n_lt_/p_gt_\n\n_lt_pre_gt_\n_lt_code_gt_Exporter version_dd_ 5.6.2_co_ Blender version_dd_ 2.79 (sub 0)\n_eq__eq__eq__eq__eq__eq__eq__eq__eq_ Conversion from Blender to Babylon.js _eq__eq__eq__eq__eq__eq__eq__eq__eq_\n\tScene settings used_dd_\n\t\tselected layers only_dd_  false\n\t\tinline textures_dd_  false\n\t\tPositions Precision _dd_  4\n\t\tNormals Precision   _dd_  3\n\t\tUVs Precision       _dd_  3\n\t\tVert Color Precision_dd_  3\n\t\tMat Weight Precision_dd_  2\n\t\ttexture directory_dd_  D_dd_\\Blender tutorial\\My models\\Babylonjs Exports\\\n\tPython World class constructor completed\n_eq__eq__eq__eq__eq__eq__eq__eq__eq_ An error was encountered _eq__eq__eq__eq__eq__eq__eq__eq__eq_\n  File _qt_C_dd_\\Users\\mandim.ICS\\AppData\\Roaming\\Blender Foundation\\Blender\\2.79\\scripts\\addons\\babylon-js\\json_exporter.py_qt__co_ line 103_co_ in execute\n    mesh _eq_ Mesh(object_co_ scene_co_ self)\nERROR_dd_  __init__() missing 3 required positional arguments_dd_ _t_forcedParent_t__co_ _t_nameID_t__co_ and _t_exporter_t_\n_eq__eq__eq__eq__eq__eq__eq__eq__eq_ end of processing _eq__eq__eq__eq__eq__eq__eq__eq__eq_\nelapsed time_dd_  0 min_co_ 0.004 secs_lt_/code_gt__lt_/pre_gt_\n\n_lt_p_gt_\n\tWhat I_t_m doing wrong?\n_lt_/p_gt_\n\n_lt_p_gt_\n\tAttached is the .blend file.\n_lt_/p_gt_\n\n_lt_p_gt_\n\tThank you in advance...\n_lt_/p_gt_\n\n_lt_p_gt_\n\t_lt_a class_eq__qt_ipsAttachLink_qt_ href_eq__qt_http_dd_//www.html5gamedevs.com/applications/core/interface/file/attachment.php?id_eq_17773_qt_ data-fileid_eq__qt_17773_qt_ rel_eq__qt__qt__gt_ExampleForBabylonjs.blend_lt_/a_gt_\n_lt_/p_gt_\n\n\n\t\t\t\n\t\t_lt_/div_gt_\n\n\t\t_lt_div class_eq__qt_ipsI_qt__gt__lt_/div_gt__lt_/div_gt_"},{"Owner":"mandim","Date":"2018-04-03T08:18:08Z","Content":"_lt_div class_eq__qt_mages_qt__gt_\n\t\t\t\n_lt_p_gt_\n\tI didn_t_t know that I had to restart blender in order to work...\n_lt_/p_gt_\n\n_lt_p_gt_\n\tSorry for the mistake.\n_lt_/p_gt_\n\n\n\t\t\t\n\t\t_lt_/div_gt_\n\n\t\t_lt_div class_eq__qt_ipsI_qt__gt__lt_/div_gt__lt_/div_gt_"}]